python/paddle/fluid/tests/unittests/ir/inference/test_mkldnn_fc_elementwise_add_fuse_pass.py

```diff
@@ -56,26 +56,22 @@ def generate_fc_bias():
             },
             outputs={"Out": ["fc_output"]},
             attrs={
-                "use_mkldnn": True,
+                "use_mkldnn": False,
```

Comment:
Are you sure that this attribute should be False?

Reply:
I tested both scenarios:
- if `fc_mkldnn_op` with given attributes is fused correctly,
- if base `fc_op` is fused correctly with sequence of `fc_mkldnn_pass` and `fc_elementwise_add_mkldnn_fuse_pass`.

Anyway, now I think that it's unnecessary and I will change it back to `use_mkldnn: True`